### PR TITLE
ci(actions): update docker login creds to use the GITHUB_TOKEN associated with the actions job

### PR DESCRIPTION
**Pull request by alex-binet**

### CODE REVIEW

**Review by Chadiii — APPROVED** [review]
seems good to me

File: Dockerfile [hunk 1]
```diff
@@ -10,11 +10,11 @@ RUN go mod download
 COPY . .
 
 ARG VERSION
-ENV VERSION $VERSION
+ENV VERSION=$VERSION
```
Comment (vincentabtasty) [inline Dockerfile:13]: Why the switching from space-separated syntax to "=" here ?

Comment (alex-binet) [reply Dockerfile:13]: The space separated syntax is considered as legacy format
https://docs.docker.com/reference/build-checks/legacy-key-value-format/

Comment (vincentabtasty) [reply Dockerfile:13]: Indeed ! Thanks for the info :+1:

**Review by vincentabtasty — APPROVED** [review]
Ok for me too ! I think GITHUB_TOKEN will do the trick :+1:

**Pull request merged**